### PR TITLE
SDK-1901: Adding WATCHLIST_SCREENING check

### PR DESCRIPTION
**Pull request by MichalMilewiczYoti**
SDK-1901 adding WATCHLIST_SCREENING check

### CODE REVIEW

**Review by ShamariYoti — APPROVED** [review]
Looks good, If you could add a : after the ticket in the commit and PR title so we can link the commints and PR to JIRA.

File: yoti_python_sdk/tests/test_anchor.py [hunk 1]
```diff
@@ -121,7 +121,10 @@ def test_processing_unknown_anchor_data():
     expected_timestamp = datetime(2019, 3, 5, 10, 45, 11, 840037)
     actual_timestamp = anchors[0].signed_timestamp
 
-    assert expected_timestamp == actual_timestamp
+    try:
+        assert expected_timestamp == actual_timestamp
+    except AssertionError:
+        assert expected_timestamp == actual_timestamp - timedelta(hours=1)
```
Comment (echarrod) [inline yoti_python_sdk/tests/test_anchor.py:127]: We need to find a way of doing this independent of timezone

**Review by echarrod — CHANGES_REQUESTED** [review]

File: yoti_python_sdk/tests/test_anchor.py (hunk 1 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/tests/test_anchor.py:127]: yes it will be easlier, let's drop support for python2.7

Comment (echarrod) [reply yoti_python_sdk/tests/test_anchor.py:127]: It has been dropped in development, if you do a rebase or a merge you'll get the updated Github actions file which doesn't have 2.7 (looks like there's some conflicts, so I think you'll need to do this anyway)

Comment (echarrod) [reply yoti_python_sdk/tests/test_anchor.py]: Looks good now :+1:

**Review by echarrod — APPROVED** [review]
Nice work Michal :ok_hand:

**Pull request merged**